src/services/user.js

```diff
@@ -0,0 +1,62 @@
+const User = require('../models/user')
+const mongoose = require('mongoose')
+const validator = require('validator')
+const { body, validationResult } = require('express-validator')
+
+// exports.validate = [
+//     body('email').isEmail(),
+  
+//     body('password').isLength({ min: 7 })
+// ]
+exports.validate = (method) => {
```

Comment:
To validate name, email, and password

Reply:
@KayedObeidat validation functions should be in the middleware or validation folder NOT in the services. Services are for DB & external APIs calls only.